DIPs/DIP1xxx-rval_to_ref.md

```diff
@@ -0,0 +1,345 @@
+# `ref T` accepts r-values
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Manu Evans (turkeyman@gmail.com)                                |
+| Status:         | Draft                                                           |
+
+## Abstract
```

Comment:
I think the abstract as written is too dense and too detailed. It can be made much more concise, likely stripped down to a single paragraph. For example, you don't need to iterate here any of the specific issues you detail in later in the Rationale. Just get across the main ideas: the current status quo is sometimes troublesome and this DIP proposes a means to alleviate the pain.

Reply:
Refactored